### PR TITLE
feat(errors): improve metrics and style of the Working error

### LINKED ISSUE #4866: Add a flow event for "working..." errors
we have error flow events for server responses but not for the working error

DataDog says that "Working..." errors are at ~ 7%, we need to confirm that this is true using flow events

### LINKED ISSUE #5354: Change "Working" from red to blue
Right now "Working..." is displayed after 10 seconds... We discussed that maybe this is a bit too quick...

@ryanfeeley - thoughts?

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #5354
Fixes #4866 

image

image

### CODE REVIEW

File: app/scripts/views/base.js [hunk 1]
```diff
@@ -86,6 +86,15 @@ define(function (require, exports, module) {
     var translated = this.translateError(err);
 
     var $error = this.$('.error');
+
+    if (err.errno && err.errno === 1005) {
```
Comment (vbudhram) [inline app/scripts/views/base.js]: Should this reference the working errno constant?

Comment (vladikoff) [reply app/scripts/views/base.js]: good idea!

Comment (shane-tomlinson) [reply app/scripts/views/base.js]: You can do `AuthErrors.is(err, 'WORKING')`

Comment (shane-tomlinson) [reply app/scripts/views/base.js]: You can do AuthErrors.is(err, 'WORKING')

**Pull request merged**